### PR TITLE
gh #179 l3 ds video device test doc

### PR DESCRIPTION
**Pull request by KarthikeyanR470**
Added L3 test document for dsVideoDevice component.

### CODE REVIEW

File: docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+Below are top test use-case for the Video Display.
+
+|#|Test-case|Description|HAL APIs|Source|Sink|
+|-|---------|-----------|--------|------|----|
+|1|Verify the Video Display framerate change with pre-change callback.|Play any video content and check the callback is triggered before when the framerate of a display changes|`dsRegisterFrameratePreChangeCB()`|`NA`|`Y`|
+|2|Verify the Video Display framerate change with post-change callback|Play any video content and check the callback is triggered after when the framerate of a display changes|`dsRegisterFrameratePostChangeCB()`|`NA`|`Y`|
+|3|Set and verify the Zoom mode of the source device|Set the Zoom mode and verify the selected Zoom mode|`dsSetDFC()`, `dsGetDFC()`|`Y`|`NA`|
+|4|Select the Device Frame Rate of Sink device |Select the Device Frame Rate of Sink device and verify|`dsSetDisplayframerate()`|`NA`|`Y`|
+|5|Set and verify the `FRF` mode|Select the`FRF`mode and verify the selected `FRF` mode|`dsSetFRFMode()`,`dsGetFRFMode()`|`NA`|`Y`|
+
```
[the remark this answers is not in the document]
Comment (KarthikeyanR470) [reply docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md:48]: Updated PR as requested.

[the remark this answers is not in the document]
Comment (KarthikeyanR470) [reply docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md:48]: Done

File: docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+- **dsVideoDevice**
+  - This is test helper class which communicates with the `L3` C/C++ test running on the `DUT` through menu
+- **L3TestClasses**
+  - These are the L3 test case classes
+  - Each class covers the each test use-case defined in [L3 Test use-cases](#level-3-test-cases-high-level-overview) table
+
+## YAML File Inputs
+
+```mermaid
+---
```
Comment (Ulrond) [inline docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md:81]: can we fix the diagrams everyone one is incorrect, and the two diagrams where mean to the merged together.

File: docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  - For more details refer [RAFT](https://github.com/rdkcentral/python_raft/blob/1.0.0/README.md) and [example_device_config.yml](https://github.com/rdkcentral/python_raft/blob/1.0.0/examples/configs/example_device_config.yml)
+
+- **componentProfile.yaml/platformProfile.yaml**
+  - Contains component-specific configurations
+  - Contains platform wide configuration broken down into separate components
+  - Example configuration file [dsVideoDevice_Settings](https://github.com/rdkcentral/rdk-halif-test-device_settings/blob/3.0.0/profiles/sink/Sink_VideoDevice.yaml)
+
+- **testSetupConfig.yaml**
+  - This configuration file contains the list of requirements for tests to execute. Eg: Copying the streams, setting environment variables etc.
+  - Example configuration file [dsVideoDevice_L3_testSetup.yml](../../../host/tests/dsVideoDevice_L3_Tests/dsVideoDevice_L3_testSetup.yml)
```
Comment (hari22yuva) [inline docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md:103]: this link is not working

File: docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+- **testSuite.yaml**
+  - This configuration file contains the list of menu items for C/C++ L3 test running on `DUT`
+  - Example configuration file [dsVideoDevice_test_suite.yml](../../../host/tests/dsClasses/dsVideoDevice_test_suite.yml)
```
Comment (hari22yuva) [inline docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md:107]: this link is not working

**Review by hari22yuva — COMMENTED** [review]
when you commit the changes please add  gh #issuenumber in commit messages

File: docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md (hunk 3 above)
Comment (KarthikeyanR470) [reply docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md:103]: Updated as per request

File: docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md (hunk 4 above)
Comment (KarthikeyanR470) [reply docs/pages/dsVideoDevice/ds-video-device_L3_Low-Level_TestSpecification.md:107]: Updated as per request

**Review by hari22yuva — APPROVED** [review]
ship it

**Review by akhilbhas — APPROVED** [review]
Looks Good

**Review by Anbukannadhasan — APPROVED** [review]
Approved

**Pull request merged**